.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+  - "2.7"
+install:
+  - pip install flake8 docutils
+script:
+  - flake8 odgovlt.py
+  - rst2html.py --strict README.rst /dev/null
+  - flake8 tests
```

Comment:
Could you please update .travis.yml, so that tests would run automatically on push via Travis? Now, I can't make tests work on my machine.